lib/rbi/parser.rb

```diff
@@ -576,7 +576,8 @@ def node_comments(node)
       def parse_comment(node)
         loc = Loc.from_prism(@file, node.location)
         string = node.location.slice
-        if string.start_with?("#:")
+        # We also ignore RDoc directives such as `:nodoc:`. See https://docs.ruby-lang.org/en/2.1.0/RDoc/Markup.html.
```

Comment:
Sorry, I realized that I linked to a really old version of the doc and updated my comment. Do you mind updating the link to that as well?